packages/block-editor/src/components/image-link-destinations/index.native.js

```diff
@@ -9,15 +9,14 @@ import { StyleSheet } from 'react-native';
  */
 import { __ } from '@wordpress/i18n';
 import { Icon, check, chevronRight } from '@wordpress/icons';
-import { blockSettingsScreens } from '@wordpress/block-editor';
 import { usePreferredColorSchemeStyle } from '@wordpress/compose';
+import { BottomSheet, PanelBody } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
-import styles from './style.scss';
-import PanelBody from '../../panel/body';
-import BottomSheet from '../bottom-sheet';
+import styles from './style.native.scss';
```

Comment:
If I am not mistaken, we are not required to explicitly add the `native` suffix. I'm not requesting it be changed here (and re-tested), but merely sharing knowledge. 
```suggestion
import styles from './style.scss';
```